src/VisualStudio/CSharp/Impl/CSharpVSResources.resx

```diff
@@ -411,4 +411,16 @@
   <data name="NewLinesForBracesProperty" xml:space="preserve">
     <value>Place open brace on new line for properties, indexers, and events</value>
   </data>
+  <data name="SetTypeInferencePreferences" xml:space="preserve">
+    <value>Type Inference preferences:</value>
+  </data>
+  <data name="UseVarForIntrinsicTypes" xml:space="preserve">
+    <value>Use 'var' for built-in types</value>
+  </data>
+  <data name="UseVarWhenPossible" xml:space="preserve">
+    <value>Use 'var' elsewhere</value>
```

Comment:
I hate that the key and value have no relation.  At least call this UseVarElsewhere.  I want to be able to read the key in the code and know what stirng will be shown to the user.